### ISSUE TITLE
Can't drag from one sortable context to another

### ISSUE THREAD

**Issue by flooyd** [clauderic/dnd-kit#1045]
Unfortunately as this is for work, I can't share a video or screenshot.

I have a single dndcontext which holds multiple tables. Each table has a sortable context within the tbody

So it looks like this roughly

<dndcontext>

<table>
  <tbody>
    <sortablecontext>
      {children} (row component with useSortable)
    </sortablecontext>
  </tbody>
</table>

<table>
  <tbody>
    <sortablecontext>
      {children} (row component with useSortable)
    </sortablecontext>
  </tbody>
</table>

<table>
  <tbody>
    <sortablecontext>
      {children} (row component with useSortable)
    </sortablecontext>
  </tbody>
</table>

</dndcontext>

The tables are placed next to each other horizontally. When I drag a row from table 1 to table 2, by the time I'm halfway there, the drag operation is cancelled and the row returns to its original position.

Any help would be appreciated :D

**Comment by flooyd** [comment]
The onDragOver is fired right as the item is snapped back into place, and the over is the second table as it should be. I'm just not sure how to proceed. I could add the item to the top of the second table, and then require the user to drag it from there to order it in the table, but I would prefer a more seamless experience

**Comment by juniusfree** [comment]
@flooyd I have encountered the same issue and found that using the `DragOverlay` component fixes it.  https://docs.dndkit.com/api-documentation/draggable/drag-overlay

**Comment by hexwit** [comment]
Yes, @juniusfree  is right. DragOverlay must be used in such cases, otherwise dom element will not be dragged  out of its sortable context.

**Comment by clauderic** [comment]
This issue was opened against a previous version of dnd-kit. The library has since undergone a major rewrite.

It has been rebuilt from the ground up with support for **multiple frameworks** (React, Vue, Svelte, Solid, and vanilla JavaScript), improved performance, and a new API.

- **Documentation**: https://dndkit.com
- **Migration guide**: https://dndkit.com/react/migration
- **GitHub**: https://github.com/clauderic/dnd-kit

If this issue is still relevant to the latest version, please open a new issue.
